### PR TITLE
Update docs #3 - c-p endpoints

### PR DESCRIPTION
**Pull request by StoneFrog**

### CODE REVIEW

File: api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json [hunk 1]
```diff
@@ -18,8 +22,11 @@
       "adults": 4,
       "children": 2,
       "message": "Important message",
+      "locked": null,
       "created_at": "CREATED_AT",
       "updated_at": "UPDATED_AT"
     }
-  ]
+  ],
+  "meta": {
```
Comment (Fredar) [inline api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json]: hmm I am not sure there is `meta` returned for this endpoint.

btw why do we need other responses?

File: content/reference/endpoints/inquiries.md [hunk 2]
```diff
@@ -46,6 +45,7 @@ lastname         | String  | Read/Write | **Required**. Inquiry's client lastnam
 message          | Text    | Read/Write | Inquiry's message. (500 characters max)
 phone            | String  | Read/Write | Inquiry's client phone number.
 source_id        | Integer | Write      | Source id related to the Inquiry.
+rental_id        | Integer | Write      | **Only update** Id of rental related to the Inquiry
```
Comment (Fredar) [inline content/reference/endpoints/inquiries.md]: I think those ids should be moved to `id` section, thats how we started to do it for `bookings_write` http://developers.bookingsync.com/reference/endpoints/bookings/

File: content/reference/endpoints/payments.md [hunk 3]
```diff
@@ -19,18 +19,21 @@ id                  | Integer  | Read       | Payment's id.
...
 country_code        | String   | Read/Write | Payer's country code, list of codes is available at [wikipedia](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).
 currency            | String   | Read/Write | Payment's currency code, list of codes described in [enums section](/reference/enums/#currencies).
+description         | String   | Read.      | Combined descriptions of all booking payments (if managed by Bookingsync).
 kind                | String   | Read/Write | **Required**. Payment's kind, list of kinds described in [enums section](/reference/enums/#payment-kinds).
 email               | String   | Read/Write | Payer's email.
 fullname            | String   | Read/Write | Payer's fullname.
 ip                  | String   | Read       | Payer's ip.
 locale              | String   | Read/Write | Payer's preferred locale, list of locales described in [enums section](/reference/enums/#locales).
 notes               | Text     | Read/Write | Payment's additional information.
+order_id            | String   | Read       | Id of order (only for instant bookings managed by Bookingsync)
```
Comment (Fredar) [inline content/reference/endpoints/payments.md]: same for ids

File: content/reference/endpoints/fees_taxes.md [hunk 4]
```diff
@@ -20,6 +21,18 @@ created_at             | [Time](/reference/enums#formats) | Read       | Fees Ta
 updated_at             | [Time](/reference/enums#formats) | Read       | Fees Tax's update time.
 {: class="table table-bordered"}
   </div>
+  <div class="tab-pane" id="rates_write" markdown="1">
+Name                   | Type    | Read/Write | Description
+-----------------------|---------|------------|------------
+id                     | Integer | Read       | Fees Tax's id.
+-----------------------|---------|------------|------------
+tax_included_in_rate   | Boolean | Read/Write | Tax amount already included in fee if set to true.
+tax_id                 | Integer | Write      | Tax's id.
```
Comment (Fredar) [inline content/reference/endpoints/fees_taxes.md]: please move to `id` section

File: api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json (hunk 1 above)
Comment (StoneFrog) [inline api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json]: Hmm true, no meta for this.
Regarding other responses, just updated what I found. But they look identical. If anything when creating inquiry with `inquiries_write_owned` it locks inquiry, and returns oly inquiries locked by this application. So the response I gave makes no sense :P

**Comment by StoneFrog** [comment]
@Fredar  updated

**Pull request merged**